gbfs.md

```diff
@@ -115,46 +129,85 @@ The following fields are all attributes within the main "data" object for this f
 
 Field Name              | Required    | Defines
 ------------------------| ------------| ----------
+stations          | Yes       | Array that contains one object per station in the system as defined below
+
 _language_              | Yes         | The language that all of the contained files will be published in. This language must match the value in the system_information file
   \- feeds               | Yes         | An array of all of the feeds that are published by this auto-discovery file
   \- name                | Yes         | Key identifying the type of feed this is (e.g. "system_information", "station_information")
   \- url                 | Yes         | Full URL for the feed
 
+
 Example:
 
 ```json
 {
   "last_updated": 1434054678,
   "ttl": 0,
+  "version":"1.0",
   "data": {
     "en": {
       "feeds": [
         {
           "name": "system_information",
-          "url": "https://www.example.com/gbfs/en/system_information"
+          "url": "https://www.example.com/gbfs/1.0/en/system_information"
         },
         {
           "name": "station_information",
-          "url": "https://www.example.com/gbfs/en/station_information"
+          "url": "https://www.example.com/gbfs/1.0/en/station_information"
         }
       ]
     },
     "fr" : {
       "feeds": [
         {
           "name": "system_information",
-          "url": "https://www.example.com/gbfs/fr/system_information"
+          "url": "https://www.example.com/gbfs/1.0/fr/system_information"
```

Comment:
Are we going to remain silent on how versioning should be handled in the URL path?

Reply:
What do you think? Should we offer a recommendation? As this is written, it's up to the discretion of the feed producers. The examples offer suggestions.

Reply:
Perhaps we should state that a URL must never change its major version. For example, if a feed doesn't want to include a version in the path, that's fine, but it must not change from v1 to v2 ever.

Reply:
One challenge for dictating versions in URL paths is that it could conflict with existing versioning schemes or numbers used by producers in URL paths. For example, HOPR currently uses integers in the URL path to identify systems - https://gbfs.hopr.city/api/gbfs/6 vs https://gbfs.hopr.city/api/gbfs/8.

So there could be significant backwards compatibility concerns.

If we require the auto-discovery file (see proposal at https://github.com/NABSA/gbfs/pull/189), I'm inclined to let producers manage their own URL paths.

Reply:
@jcn I just pushed a change that responds to what you pointed out. Please let me know what you think…